filtro/templates/filtro/filtros.html

```diff
@@ -21,19 +21,56 @@
+                                    <td>
+                                        <div class="progress progress-xs margin-vertical-10" id="progresso_{{item.id}}" style="display:none;">
+                                            <div class="progress-bar progress-bar-danger" id="barra_{{item.id}}"></div>
+                                        </div>
+                                    </td>
                                     <td class="text-nowrap">        
                                         <a href="{% url 'filtros-filtro' item.id %}" data-toggle="tooltip" data-original-title="Editar"><i class="fa fa-pencil text-inverse m-r-10"></i></a>
-                                        <a href="#" data-toggle="modal" data-target="#confirmarexclusaomodal" data-original-title="Excluir" onclick="atualizar_filtro_exclusao('{{item.id}}', '{{item.nome}}')"><i class="fa fa-close text-danger m-r-10"></i></a>
-                                        <a href="#" data-toggle="tooltip" data-original-title="Executar Filtro"> <i class="fa fa-bolt text-success m-r-10"></i> </a>
-                                        <a href="#" data-toggle="tooltip" data-original-title="Baixar Resultados"> <i class="fa fa-download text-warning m-r-10"></i></a>
+
+                                        <a href="#" data-toggle="modal" data-target="#confirmarexclusaomodal" data-original-title="Excluir" 
+                                            onclick="atualizar_filtro_exclusao('{{item.id}}', '{{item.nome}}')">
+                                            <i class="fa fa-close text-danger m-r-10"></i>
+                                        </a>
+
+                                        <a href='{% url "filtros-classificar" item.id %}'
+                                            id="btnexecutar_{{item.id}}"
+                                            data-toggle="tooltip" data-original-title="Executar Filtro"> 
+                                            <i class="fa fa-bolt text-success m-r-10"></i>
+                                        </a>
+
+                                        <a href="{% url "filtros-compactar-resultado" item.id %}"
+                                            id="btncompactar_{{item.id}}" 
+                                            data-toggle="tooltip" data-original-title="Compactar Resultados" 
+                                            style="display:none;"> 
+                                            <i class="fa fa-file-zip-o text-warning m-r-10"></i>
+                                        </a>    
+
+                                        <a href="#"
+                                            id="btnbaixar_{{item.id}}" 
+                                            data-toggle="tooltip" data-original-title="Baixar Resultados" 
+                                            style="display:none;"> <i class="fa fa-download text-warning m-r-10"></i>
+                                        </a>
+
+
```

Comment:
Just an aesthetic note, but this has more separations than previous blocks. Unintended.